ssh_check/tests/test_ssh.py

```diff
@@ -0,0 +1,80 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import threading
+
+import pytest
+from datadog_checks.stubs import aggregator
+
+from datadog_checks.ssh_check import CheckSSH
+
+
+@pytest.fixture
+def Aggregator():
```

Comment:
every other test uses `aggregator` as the name for the fixture, we should eventually move this to a common testing-support package so not a big deal but:
1) I personally don't like uppsercase function names
2) in this case, we provide a fixture for a module called `aggregator` so it's more explicit if the fixture has the same name

Reply:
I'll change this now.

I was trying to avoid the import on every fixture call.

Reply:
we can use an alias for that, even if performance wise I don't think we get anything since the module should be cached after the first call to the fixture func